src/Compilers/CSharp/Portable/CodeGen/EmitOperators.cs

```diff
@@ -225,6 +225,10 @@ private void EmitBinaryOperatorInstruction(BoundBinaryOperator expression)
                     }
                     break;
 
+                case BinaryOperatorKind.UnsignedRightShift:
```

Comment:
Consider adding an assertion in `IsUnsignedBinaryOperator` that it's not meant to be used on an unsigned right shift.

Reply:
> Consider adding an assertion in ```IsUnsignedBinaryOperator``` that it's not meant to be used on an unsigned right shift.

Will add in one of the next PRs